libs/wingsdk/src/std/json.ts

```diff
@@ -453,4 +449,31 @@ export class MutJson {
   public tryAsBool(): boolean | undefined {
     throw new Error("Macro");
   }
+
+  /**
+   * Removes the specified element from a map.
+   *
+   * @macro (delete ($self$)[$args$])
+   *
+   * @param key The key
+   * @returns true if the given key is no longer present
+   */
+
+  public delete(key: string): boolean {
+    key;
+    throw new Error("Macro");
+  }
```

Comment:
What happens if `delete()` is called on something that isn't a map? like `Json true` or `Json [1, 2, 3]`? Can we add tests for this?

Reply:
It should return true (matching the return comment). 
null is also technically valid json and that would throw here (bad), but most of our json methods wouldn't correctly handle null anyways.
Agree a test would be good, my b for approving without more tests.

Reply:
I could try adding extra tests for some of those edge cases in another PR!